### ISSUE TITLE
Page does not start with a level 1 heading

### ISSUE THREAD

**Issue by libmanuk** [uklibraries/UKL_HeaderFooter#43]
Header levels 4 and 5 have h2 elements.  These should be switched to h1 elements in follow accessibility best practices.  This issue was flagged via SiteImprove analysis.
